src/podofo/main/PdfXMPPacket.cpp

```diff
@@ -8,6 +8,7 @@
 #include "PdfXMPPacket.h"
 #include <podofo/private/XmlUtils.h>
 #include <libxml/xmlsave.h>
+#include <libxml/parser.h>
```

Comment:
this is needed because `xmlReadMemory` and `XML_PARSE_NOBLANKS` is no longer exposed from the only libxml include here (xmlsave.h no longer includes parser.h, which included missing symbols)

although seems to be coming from somewhere deeper, I just added the include for parser.h here which defines both missing symbols.